core/libp2p/CMakeLists.txt

```diff
@@ -8,5 +8,15 @@ add_subdirectory(transport)
 add_subdirectory(crypto)
 add_subdirectory(peer)
 add_subdirectory(muxer)
-add_subdirectory(upgrader)
 add_subdirectory(protocol_muxer)
+
```

Comment:
Is it supposed to be used with host builder or forgotten to remove.

Reply:
it is supposed to be used as a top level library, but currently it does not compile because host builder is not implemented